### ISSUE TITLE
Make length value more readable.

### ISSUE THREAD

**Issue by Carlgo11** [hoovercj/perfectpass#3]
Hi @hoovercj,

I like your site! But it's a bit too cody looking for normal people to understand it I think.
1. Length in an array? Maybe it would be better to put 2 rows for length instead? 1 for min value and one for max.
2. "Non-ascii" on Gmail. I don't think that many people know what ascii means. For example I recently had to explain what unicode is.

3. Must use N of these types. That's a very programmy way of saying it. Should probably be changed.

**Comment by Walkman100** [comment]
:+1:
I was thinking pretty much the same thing when I had a look, just was not sure how to explain

**Comment by hoovercj** [comment]
I totally agree. Though at first I was confused by you saying 'too cody looking' because that's my name XD

1. I'm not the biggest fan of the interval notation that I'm using, but I also feel like two columns are a waste of space. Right now the table is about as wide as it can be without overflowing at half-screen. Maybe just doing something like `8-32` instead of `[8, 32] would be sufficient. What do you think?

2. I'm totally fine with removing "non-ascii" as we get more site information. It's something that @curtisullerich found and at this point I'm erring on the side of keeping any potentially relevant information until I have a better idea of what is normal for sites.

3. It's hard to efficiently display the information about required types. Does the password require uppercase? lowercase? symbols? If it were as easy as a 'yes' or 'no' answer, we could just have dedicated columns. But PayPal doesn't require them all, it just requires that you have at least two of the three.

One alternative to the current method: Maybe the column heading should say `Required character types` and the ones like verified by visa that require two specific types, just have the text or icons for those types (i.e. `letters, numbers`) while PayPal could say '2 types from: letters, numbers, special` 

I'd really like to get icons in there instead of the text though. Or at least see what that would look like.

**Comment by hoovercj** [comment]
Commit c1d2755bfb0f7528cf5d89a3a58e009a6bdfe617 has some changes.

1. Made the change from `[min, max]` to `min - max`
2. Moved "non-ascii" to the expanded information
3. Columns present information in format `Requires 2: uppercase, lowercase, special`

Commit c1db4c141d9b8179563f7661f85e2c3200c5248f also has a change

* Removed `n/a` from columns without information. If there is no length requirement, banned chars, required chars, etc. it no longer has `n/a` and just has a blank column. I think it looks cleaner this way.
